src/broadcast.jl

```diff
@@ -165,3 +165,68 @@ end
         return dest
     end
 end
+
+# Work around for https://github.com/JuliaLang/julia/issues/27988
+# The following code is borrowed from https://github.com/JuliaLang/julia/pull/43322
+# with some modification to make it also works on 1.6.
+# TODO: make `broadcast_flatten` call `Broadcast.flatten` once julia#43322 is merged.
+module StableFlatten
+
+export broadcast_flatten
+
+using Base: tail
+using Base.Broadcast: isflat, Broadcasted
+
+maybeconstructor(f) = f
+maybeconstructor(::Type{F}) where {F} = (args...; kwargs...) -> F(args...; kwargs...)
```

Comment:
Is that for broadcasting things like `Float64.(SVector{2}([1, 2]))`? If so then maybe you could add such test and we'd have full test coverage.

Reply:
done